exchanges/binance/binance_wrapper.go

```diff
@@ -636,6 +636,21 @@ func (b *Binance) UpdateAccountInfo(assetType asset.Item) (account.Holdings, err
 			})
 		}
 
+		acc.Currencies = currencyDetails
+	case asset.Margin:
+		accData, err := b.GetMarginAccount()
+		if err != nil {
+			return info, err
+		}
+		var currencyDetails []account.Balance
+		for i := range accData.UserAssets {
+			currencyDetails = append(currencyDetails, account.Balance{
+				CurrencyName: currency.NewCode(accData.UserAssets[i].Asset),
+				TotalValue:   accData.UserAssets[i].Free + accData.UserAssets[i].Locked,
+				Hold:         accData.UserAssets[i].NetAsset,
```

Comment:
NetAsset -> Locked

Reply:
I may be incorrect but from the API docs, it looks like NetAsset == Free - any borrowed amounts.  Another account pass will have to be done to integrate borrowed amounts and the differentiation between net.

Reply:
Sorry for the late reply. Have been focusing on other business.
I am no expert on how Binance accounts for margined positions so I changed the code in accordance with your advice.

I implemented `historicalTrades  = "/api/v3/historicalTrades"` REST endpoint, which queries a slice of public trades on that
pair.

- [x] Tested in non-mocking environment